how-to/security/apparmor.md

```diff
@@ -225,6 +211,43 @@ Profiles are meant to provide security and so can't be too permissive. But often
   * To mitigate the drawbacks of above approaches, **local includes** were introduced, adding the ability to write arbitrary rules that not run into issues during upgrades that modify the packaged rule.
   * The files can be found in `/etc/apparmor.d/local/` and exist for the packages that are known to sometimes need slight tweaks for special setups.
 
+
+## Disabling or Re-enabling AppArmor
+
+Starting with Ubuntu 24.04 and later, the AppArmor services are baked into the Ubuntu Kernel, and require special steps to fully disable or re-enable.
```

Comment:
You and me know what you mean, but for others it might be "which are the *special* steps".
Maybe:

```suggestion
Starting with Ubuntu 24.04 and later, the AppArmor services are baked into the Ubuntu Kernel. While before you disable it by not loading the service, it now requires setting a module parameter on the kernel command line to fully disable or re-enable.
```